101-event-grid-cloudevents/azuredeploy.json

```diff
@@ -0,0 +1,189 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "metadata": {
+        "description": "The region where resources are deployed"
+      },
+      "defaultValue": "[resourceGroup().location]"
+    },
+    "logicAppName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Logic App"
+      }
+    },
+    "eventGridTopicName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Event Grid Topic"
+      }
+    },
+    "eventGridTopicSchema": {
+      "type": "string",
+      "metadata": {
+        "description": "Event Grid Topic schema"
+      },
+      "allowedValues": [
+        "EventGridSchema",
+        "CloudEventV01Schema"
+      ],
+      "defaultValue": "CloudEventV01Schema"
+    },
+    "eventGridSubscriptionName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Event Grid Subscription"
+      }
+    },
+    "eventGridSubscriptionDeliverySchema": {
+      "type": "string",
+      "metadata": {
+        "description": "Event Grid Subscription delivery schema"
+      },
+      "allowedValues": [
+        "EventGridSchema",
+        "CloudEventV01Schema"
+      ],
+      "defaultValue": "CloudEventV01Schema"
+    },
+    "eventGridSubscriptionEndpointType": {
+      "type": "string",
+      "metadata": {
+        "description": "Event Grid Subscription endpoint type"
+      },
+      "allowedValues": [
+        "Webhook"
+      ],
+      "defaultValue": "Webhook"
+    },
+    "eventGridSubscriptionIncludedEventTypes": {
+      "type": "string",
+      "metadata": {
+        "description": "Comma delimited list of filters for Event Grid Subscription"
```

Comment:
is there an allowed list here or is it open ended?  can we add to the description?

Reply:
@bmoore-msft It's actually open-ended. Default is `all`. I added more description for clarity.